packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/exposureoffset/ExposureOffsetValue.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.exposureoffset;
+
+/**
+ * This represents the exposure offset value. It holds the minimum and maximum values, as well as
+ * the current setting value.
+ */
+public class ExposureOffsetValue {
```

Comment:
Why are min/max and current value the same object? That seems surprising since they are fundamentally different kinds of values.

Reply:
You are correct. The `ExposureOffsetValue` is currently used to return all values at once to the `Camera` class. However it would make much more sense if the `Camera` class could just call the `ExposureOffsetFeature.getMinExposureOffset` or `ExposureOffsetFeature.getMaxExposureOffset` methods when it needs to returns these values to the Dart side.

If have removed the `ExposureOffsetValue` and replaced it with a simple double and made the `getMinExposureOffset` and `getMaxExposureOffset` public.